### PR TITLE
[zh] Resync security overview

### PR DESCRIPTION
**Pull request by tengqm**
This PR resyncs the security overview page from English site and it also fixes a link.

closes: #27214
supersedes: #27216
coauthor-by: @vaibhav2107

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
LGTM

File: content/zh/docs/concepts/security/overview.md [hunk 1]
```diff
@@ -209,7 +210,7 @@ Disallow privileged users | When constructing containers, consult your documenta
 容器漏洞扫描和操作系统依赖安全性 | 作为镜像构建的一部分，您应该扫描您的容器里的已知漏洞。
 镜像签名和执行 | 对容器镜像进行签名，以维护对容器内容的信任。
 禁止特权用户 | 构建容器时，请查阅文档以了解如何在具有最低操作系统特权级别的容器内部创建用户，以实现容器的目标。
-
+使用带有较强隔离能力的容器运行时 | 选择提供较强隔离能力的[容器运行时类](/zh/docs/concepts/containers/runtime-class/).
```
Comment (ydFu) [inline content/zh/docs/concepts/security/overview.md:213]:
```suggestion
使用带有较强隔离能力的容器运行时 | 选择提供较强隔离能力的[容器运行时类](/zh/docs/concepts/containers/runtime-class/)。
```

**Comment by ydFu** [comment]
/lgtm
thank for @tengqm,@vaibhav2107

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**